static/app/components/events/interfaces/frame/deprecatedLine.tsx

```diff
@@ -486,7 +486,7 @@ const NativeLineContent = styled('div')<{isFrameAfterLastNonApp: boolean}>`
 
 const DefaultLine = styled('div')`
   display: grid;
-  grid-template-columns: 1fr auto auto;
+  grid-template-columns: 1fr auto ${space(2)} + ${space(1)}; /* sm icon size + left margin */
```

Comment:
@17hogeju So this seems to work, but instead of adding the `space(1)` to account for the toggle button's left margin, I'd rather remove the `margin-left` from the `ToggleContextButton` styles.

It looks like if you do that, the toggle button does not fit snugly to the right side, however! You can use `justify-self` to change the justification to `end` for the toggle button to correct that.

Reply:
> Also fyi - the + here doesn't do what you think it should do, since it is outside of the ${}. The + gets stripped out (since it is not valid css) and it adds an additional column

Makes sense. I thought that might be the case.